db/migrate/20191005223221_add_details_to_tests.rb

```diff
@@ -0,0 +1,7 @@
+class AddDetailsToTests < ActiveRecord::Migration[6.0]
+  def change
+    change_column_default(:tests, :level, from: false, to: 0)
```

Comment:
аналогично предыдущему:
```suggestion
    change_column_default(:tests, :level, from: nil, to: 0)
```